Videoupload

wagnerexpositio opened the pull request.

k-nut commented:
@lisaq I think we can ignore codecov complaining here. We don't really need tests for the command I think. Or would you disagree?

k-nut commented:
Todo: Add some example tags

wagnerexpositio committed:
get presigned url from minio s3

wagnerexpositio committed:
adding videoupload backend function

wagnerexpositio committed:
adding new widget to backend

dev7ch committed:
Adjust minio presigned url endpoint

- use `presigned_put_object` method to allow direct upload via PUT
- simplify return, obj to string
- remove content type from response

dev7ch committed:
Update models.py

Proposal of minio pre signed url handling in admin ui.

- use a slug  which contains the word `video`
- compare slug in models and generate pre_signed url on request
- output as file field with download link in form responses

### why?
Currently, its a problem to use format options in json schema form due validation purposes.
A simple string text field is working fine with vadlitaionb(only the filename as string is in form submit.

This and #21  ist working with the current frontend integrations of https://github.com/correctiv/crowdnewsroom-frontend/tree/videoupload

wagnerexpositio committed:
Merge pull request #21 from correctiv/videoupload-patch-1

Adjust minio presigned url endpoint

wagnerexpositio committed:
Merge pull request #22 from correctiv/videoupload-patch-2

Update models.py

wagnerexpositio committed:
changing structure for getting video download url
